hub/api/dataset.py

```diff
@@ -120,41 +117,36 @@ def __getitem__(
     def create_tensor(
         self,
         name: str,
-        htype: Optional[str] = None,
-        chunk_size: Optional[int] = None,
-        dtype: Optional[str] = None,
-        extra_meta: Optional[dict] = None,
+        htype: str = DEFAULT_HTYPE,
+        **kwargs,
```

Comment:
right, the reason i removed this was because we haven't really defined `htype`s yet. i totally agree we should have this in the docstring, but i want to defer this to the `htype` PR

Reply:
is that ok with you?

Reply:
@McCrearyD what am I missing here? Why should the `htype` options affect whether the docstring mentions `chunk_size`?

Reply:
because this needs higher level organization, which `htype` is part of that conversation

Reply:
i can re-add this to the docstring if either of you think i should

Reply:
Oh, I see. @McCrearyD IMO we should have a list of "special" meta documented in the docstring. We don't have to have the param (because we have kwargs) but it should be obvious from the docstring that setting chunk_size via kwarg is an option

Reply:
This is THE docstring for create_tensor. User's aren't going to want to dive into other methods inside our codebase, this is extremely user-facing and should explain itself as much as possible.

Reply:
right, can we do this when we make the `htype`s? i think it is most appropriate being added in there

Reply:
right, i will update this docstring here to have `chunk_size` and `dtype` but we should rework it with the `htype`s being defined

Reply:
done